device/bgp/bgpadvanced.yaml

```diff
@@ -45,3 +45,13 @@ components:
             If not configured, MD5 authentication will not be enabled.
           type: string
           x-field-uid: 5
+        passive_mode:
+          description: |-
+            If enabled then the BGP Finite State Machine will passively wait 
+            for the remote BGP peer to establish the BGP TCP connection. 
+            If the passive mode is true, BGP peer will wait for the remote BGP peer 
```

Comment:
I dont think this is true for passive = true . https://github.com/openconfig/public/blob/master/release/models/bgp/openconfig-bgp-common.yang#L339
I think the first part is all that passive = true should guarantee. i.e. ( If set to true, the local BGP peer will wait for the remote peer to initiate the BGP session by establishing the TCP connection , rather than initiating sessions from the local peer )

Reply:
Two more notes : If you agree and change description , please change PR description as well ( I have made some fixes for typo and code segment but left rest as is ) . Also, good to log an Issue and link it via Development option in right hand side.